### ISSUE TITLE
Tracker doesn't seem to recognise rogue decks with Deathmantle Valera is the hero skin

### ISSUE THREAD

**Issue by catdogbat** [HearthSim/Arcane-Tracker#171]
Anyone else noticed?

**Comment by AleksanderStefankowski** [comment]
Phew, so that's what it's about! I was so worried. I confirm this is the case catdogbat. Fully valid issue.

**Comment by catdogbat** [comment]
Yeah I thought the time had come for it to stop working on my phone but thankfully it's just the hero!

**Comment by mariodmmariodm** [comment]
Hi,

The problem is with all the new skins won in the rewards track!

Bye,
Mario

Il Ven 7 Mag 2021, 09:22 catdogbat ***@***.***> ha scritto:

**Comment by catdogbat** [comment]
I only have rogue and Arcanist Jaina. I tried Arcanist Jaina and that one seems to work ok!
